src/views/somns.R

```diff
@@ -387,17 +391,22 @@ perf_diff_table_es <- function(data_es, stats_es, warmup_es, profiles_es, start_
   # b <- "DeltaBlue"
   # data_ea <- data_b
 
-  for (b in levels(data_es$bench)) { data_b <- data_es %>% filter(bench == b) %>% droplevels()
+  for (b in levels(data_es$bench)) { data_b <- data_es %>% filter(bench == b) %>% droplevels() # nolint
```

Comment:
We are not running R linting in CI.
Should be add it?
Otherwise, I'd drop the comment.